byterun/weak.c

```diff
@@ -400,21 +501,40 @@ CAMLprim value caml_ephe_blit_key (value ars, value ofs,
       do_set (ard, offset_d + i,  Field (ars, offset_s + i));
     }
   }
+}
+
+CAMLprim value caml_ephe_blit_key (value ars, value ofs,
+                                   value ard, value ofd, value len)
+{
+  if (Long_val(len) == 0) return Val_unit;
+  mlsize_t offset_s = raise_if_invalid_offset(ars, ofs, "Weak.blit");
+  mlsize_t offset_d = raise_if_invalid_offset(ard, ofd, "Weak.blit");
+  mlsize_t length = Long_val (len);
```

Comment:
This is arguable, because this is actually the C stub called by Weak.blit. This stub is shared with the ephemeron's.